solutions/01_hello_html/img.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <body>
+    
+    <img src = "http://weknowyourdreams.com/image.php?pic=/images/dog/dog-07.jpg">
+    <p>It's a dawg</p>
+    
+    <img src = "http://bebusinessed.com/wp-content/uploads/2014/03/734899052_13956580111.jpg">
```

Comment:
very nice! just make sure `src="..."` doesnt have extra spaces before/after the equals (`=`) sign